sdks/python/apache_beam/runners/portability/fn_api_runner/execution.py

```diff
@@ -367,6 +413,73 @@ def _build_process_bundle_descriptor(self):
         state_api_service_descriptor=self.state_api_service_descriptor(),
         timer_api_service_descriptor=self.data_api_service_descriptor())
 
+  def commit_output_views_to_state(self):
```

Comment:
Not sure what "output views" means. Maybe call this commit_side_inputs_to_state as well?

Reply:
done